model-optimizer/extensions/middle/UpsampleToResample.py

```diff
@@ -92,10 +92,10 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
 
         if input_shape_rank == 4:
             begin_value = int64_array([get_height_dim(layout, input_shape_rank)])
-            factor_value = np.array([height_scale, width_scale])
+            factor_value = int64_array([height_scale, width_scale])
```

Comment:
are `height_scale` and `width_scale` always integer? I am afraid that you can round them.

Reply:
Fixed.